File: LicentaAngular/src/app/models/series-model.ts
import { Genre } from "./genre";
import { Rating } from "./rating";

export interface SeriesModel {
    id: string;
    title: string;
    director: string;
    description: string;
    releaseDate: Date;
    prequelId?: string;
    sequelId?: string;
    genre: Genre;
    rating: Rating;
    nrEps: number;
    picture: string;
    prequelTitle?: string;
    sequelTitle?: string;
}

File: LicentaUI/Models/SeriesModel.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace LicentaUI.Models
{
    public class SeriesModel
    {
        public string ID { get; set; }

        [Display(Name = "Series Titles")]
        public string Title { get; set; }

        public string Director { get; set; }

        public string Description { get; set; }

        public DateTime RelaseDate { get; set; }

        public string PrequelID { get; set; }

        public string SequelID { get; set; }

        public Genre Genre { get; set; }

        public Rating Rating { get; set; }

        public int NrEps { get; set; }

        public string Picture { get; set; }
    }
}